pkg/api/handlers/compat/images_build.go

```diff
@@ -140,7 +140,6 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		Registry:      "docker.io",
 		Rm:            true,
 		ShmSize:       64 * 1024 * 1024,
-		TLSVerify:     true,
```

Comment:
Okay now I see that it additionally checks the existence of `"tlsVerify"` query parameter below in the code:

```go
if _, found := r.URL.Query()["tlsVerify"]; found {
	systemContext.DockerInsecureSkipTLSVerify = types.NewOptionalBool(!query.TLSVerify)
	systemContext.OCIInsecureSkipTLSVerify = !query.TLSVerify
	systemContext.DockerDaemonInsecureSkipTLSVerify = !query.TLSVerify
}
```

But does it really break anything? I see, setting a default value just doesn't have any effect cause the parameter only used when it's been actually assigned some value.

Maybe just keep it for documenting purposes? After all it says in docs that it's true by default.

Reply:
You are absolutely right.  I shouldn't have checked it without a fresh coffee.

` r.URL.Query()["tlsVerify"]` is ignoring the default we set.

> Maybe just keep it for documenting purposes? After all it says in docs that it's true by default.

Yes, I am OK with keeping it as is.  The default is set further down the stack in any case but it doesn't hurt.  As you mentioned, at least for docs.